config/application.rb

```diff
@@ -175,3 +175,4 @@ class Application < Rails::Application
 end
 
 require 'samson/hooks'
+require "#{Rails.root}/lib/logging"
```

Comment:
can this be `require 'logging'` ?
maybe make it samson_logging so it does not override others by accident ...